ram/sram.asm

```diff
@@ -215,10 +217,9 @@ s4_a03b:: ds 37 * 40
 s4_a603:: ds 8
 
 s4_a60b:: db
-s4_a60c:: db
-s4_a60d:: db
+s4_a60c:: dw
 
-s4_a60e:: dw
+sCrystalFlags:: ds 2
```

Comment:
`wCrystalFlags` is one byte, right? So this can be `sCrystalFlags:: db`, and change the `ds $1f0` below to `$1f1`.